### PR TITLE
log: Add comments to logger interfaces

### LINKED ISSUE #262: Document [log] interfaces
### Location
`[log] StdLogger, LevelLogger, Logger…`

### Problem
Interface functions are missing doc.

### Proposal
Add the docs.

### PR DESCRIPTION
**Pull request by matthiasgeihs**
Closes #262

### CODE REVIEW

File: log/log.go [hunk 1]
```diff
@@ -51,34 +51,58 @@ var _ StdLogger = &log.Logger{}
...
 	Trace(...interface{})
+	// Debug logs a message at level Debug.
 	Debug(...interface{})
+	// Info logs a message at level Info.
 	Info(...interface{})
+	// Warn logs a message at level Warn.
 	Warn(...interface{})
+	// Error logs a message at level Error.
 	Error(...interface{})
+	// Traceln logs a message at level Trace.
```
Comment (manoranjith) [inline log/log.go:97]: Add `Arguments are handled in the manner of fmt.Println.`

**Review by manoranjith — CHANGES_REQUESTED** [review]
Have one suggestion.

**Comment by matthiasgeihs** [comment]
Addressed your comment. Please have another look and feel free to merge.

**Comment by matthiasgeihs** [comment]
rebased

**Review by manoranjith — APPROVED** [review]

**Pull request merged**